### PR TITLE
Bug Fix - Nav elements overlap

### PR DESCRIPTION
**Pull request by chetan-187**
fixes https://github.com/supabase/supabase/pull/25966

## I have read the [CONTRIBUTING.md](https://github.com/supabase/supabase/blob/master/CONTRIBUTING.md) file.

YES

## What kind of change does this PR introduce?
Bug fix - CSS issues in the side navigation panel of the project dashboard page: When the side panel is expanded and scrolled - supabase logo appears at the bottom layer and other items are layered above it.

## What is the current behavior?
image

[ISSUE LINK](https://github.com/supabase/supabase/issues/23225)

## What is the new behavior?
image

### CODE REVIEW

**Comment by chetan-187** [comment]
This is my first time contributing to an open-source project! I reviewed the contribution guidelines ([CONTRIBUTING.md](https://github.com/supabase/supabase/blob/master/CONTRIBUTING.md)) after submitting my pull request, and I noticed there's already another PR open for the same issue. I understand if mine might be redundant, but I'd be thrilled if you could still consider it. 
**Having this contribution accepted would be a huge motivator for me to keep getting involved in the community.**

**Comment by saltcod** [comment]
https://github.com/supabase/supabase/issues/26763

**Comment by saltcod** [comment]
this LGTM — @MildTomato for a +1 please!

**Comment by chetan-187** [comment]
Hello @MildTomato, Could you please review this once. This has been pending for a while :)

**Comment by MildTomato** [comment]
checking now @chetan-187 :D

File: apps/studio/components/layouts/ProjectLayout/NavigationBar/NavigationBar.tsx [hunk 1]
```diff
@@ -97,21 +97,21 @@ const NavigationBar = () => {
...
         )}
         onMouseEnter={() => snap.setNavigationPanelOpen(true)}
         onMouseLeave={() => {
           if (!userDropdownOpen) snap.setNavigationPanelOpen(false)
         }}
       >
-        <ul className="flex flex-col gap-y-1 justify-start px-2">
-          {(!navLayoutV2 || !IS_PLATFORM) && (
+        {(!navLayoutV2 || !IS_PLATFORM) && (
+          <div className="sticky top-0 mb-1 py-1 px-2 flex h-[48px] bg-studio z-50">
```
Comment (MildTomato) [inline apps/studio/components/layouts/ProjectLayout/NavigationBar/NavigationBar.tsx:111]: z-50 is possibly wrong as z-50 and above is reserved for overlays (Dropdown menu, tooltips etc), so this nav could end up being rendered on top of overlays. z-10 is safe to use.

**Review by MildTomato — DISMISSED** [review]

File: apps/studio/components/layouts/ProjectLayout/NavigationBar/NavigationBar.tsx (hunk 1 above)
Comment (chetan-187) [reply apps/studio/components/layouts/ProjectLayout/NavigationBar/NavigationBar.tsx:111]: Thanks @MildTomato, I will check.

**Comment by chetan-187** [comment]
Hello @MildTomato & @saltcod, I completed the changes, please let me know if anything else is required here.

**Comment by Hallidayo** [comment]
Hi @supabase/frontend  - Could this one be checked as we've got a couple of issues open that relates to this issue.

**Comment by saltcod** [comment]
checked preview, looks good. thanks @chetan-187!

screenshot-2024-08-08-at-14 31 23

**Review by saltcod — APPROVED** [review]

**Pull request merged**